package.json

```diff
@@ -6,6 +6,7 @@
     "start": "npm run start:dev",
     "start:dev": "nodemon --watch src --watch forta.config.json -e js,json --exec 'forta-agent run'",
     "start:prod": "forta-agent run --prod",
+    "docker:test": "docker build -t testbuild . && docker run testbuild",
```

Comment:
I'm still uncertain why I run into this issue, but `docker build` fails on my machine unless I do `--network=host`.

I'd suggest using a 'post' script to break this one liner up. We're a little limited with how we can pass script args to `npm`:

```
    "docker:test": "docker build -t testbuild .",
    "postdocker:test": "docker run testbuild",
```

This way, I can pass `--network=host` to npm with something like `npm run docker:test -- --network=host`


Check out: https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts

Reply:
I changed "docker:test" to "test:docker" and split out the post section as requested.